meson.build

```diff
@@ -74,18 +75,38 @@ if get_option('wayland') != 'false'
   endif
 endif
 
+# WIN32
+use_win32 = false
+if get_option('win32') != 'false' and host_machine.system() == 'windows'
+  use_win32 = true
+  libva_utils_flags = [ '-DHAVE_VA_WIN32=1', '/std:c++20' ]
+  require_win32 = get_option('win32') == 'true'
+endif
+
 add_project_arguments(libva_utils_flags,
                       language: ['c', 'cpp'])
 
-subdir('common')
-subdir('decode')
-subdir('encode')
-subdir('putsurface')
+if use_win32
+  subdir('getopt')
+  win32_deps = [ dependency('libva-win32s', required: require_win32), idep_getopt ]
```

Comment:
Nits:
 - typo s/win32s/win32/
 - getopt cannot be "not_found()", so I would flatten the foreach loop below